modules/dashboard/src/components/Dashboard.js

```diff
@@ -199,12 +191,12 @@ class Dashboard extends React.Component {
                 </IconButton>
               </div>
               <Divider />
-              <List>{mainListItems}</List>
+              <SidebarLinks publicUrl={publicUrl} />
```

Comment:
Replaced the `mainListItems` constant w a react component function called `SidebarLinks` so I could pass in the url as a prop.